include/picongpu/fields/currentDeposition/Esirkepov/Esirkepov.hpp

```diff
@@ -97,8 +94,16 @@ namespace picongpu
                     constexpr bool isSupportEven = (supp % 2 == 0);
                     RelayPoint<isSupportEven>()(iStart, iEnd, line.m_pos0[d], line.m_pos1[d]);
                     gridShift[d] = iStart < iEnd ? iStart : iEnd; // integer min function
+                    bitpacking::set(
+                        status[d],
+                        bitpacking::Status::START_PARTICLE_IN_ASSIGNMENT_CELL,
+                        gridShift[d] == iStart);
+                    bitpacking::set(
+                        status[d],
+                        bitpacking::Status::END_PARTICLE_IN_ASSIGNMENT_CELL,
+                        gridShift[d] == iStart);
```

Comment:
Self-reminder to discuss offline

Reply:
condition should be `gridShift[d] == iEnd)`, I will double check this part